WebApi - Step 2 potential issue for novice developer

aguerere opened the pull request:
The using statement assumes that the WebApi project is named Api.

This can cause confusion for novice developers. Its better to display a
placeholder in the documentation, that way it is explicit to the
developer that this should be replaced with their current project name.

woloski commented:
not convinced about the title

woloski commented:
seems like this is more "Auth0 Core Concepts"

sandrinodimattia commented:
Ok makes sense. But this is really focused on the application. How about we create a new section in the left navigation: "Core Concepts". With things like: High level overview (doc which we are planning to write), Applications (this doc), Connections (with links to database, enterprise, social, password less), Rules (link to rules document)?

pose commented:
@sandrinodimattia @woloski just to prevent duplicated documents: Is this the same as https://github.com/auth0/auth0-cse/blob/master/glossary.md but publicly available?

sandrinodimattia commented:
@pose, the glossary is a big list with words and their definition. This document only covers "Applications" and is a few pages long. So let's say its covers 1 word of the glossary with a lot of detail ;)

pose commented:
@sandrinodimattia Awesome, then I'll add a link to the glossary to this. Thanks for the clarification!

Commit:
The using statement assumes that the WebApi project is named Api

This can cause confusion for novice developers. Its better to display a
placeholder in the documentation, that way it is explicit to the
developer that this should be replaced with their current project name.

aguerere commented:
I got confused and closed the pull request because of the "Merge remote-tracking branch" commit. On further inspection realized the only file changed is the intended one. Apologies.

ntotten commented:
Please rebase to avoid the merge commit

aguerere commented:
Rebase ready.
